pages/osx/xcode-select.md

```diff
@@ -0,0 +1,24 @@
+# xcode-select
+
+> Switch between different versions of Xcode and the included developer tools.
+> Also used to update the path to Xcode if it is moved after installation.
+
+- Install Xcode's command line tools:
+
+`xcode-select --install`
+
+- Select {{/path/to/Xcode.app/Contents/Developer}} as the active developer directory."
+
+`xcode-select -s {{/path/to/Xcode.app/Contents/Developer}`
```

Comment:
```suggestion
`xcode-select -s {{/path/to/directory}`
```

Reply:
Per #6 on [this page,](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md) an actual example makes more sense here. I cannot image a scenario where you would not end that path with either `Xcode.app` or `Xcode.app/Contents/Developer`. I'm not even sure it's possible to use a path that _doesn't_ fall under some `Xcode.app`.

Thoughts?

Reply:
cc @zdroid

Reply:
cc @zdroid

Reply:
I'm neutral on this, but maybe the one suggested by @schneiderl is more in line with the rest of the project.

Reply:
Maybe I should specify in the description that this should be a path to an `Xcode.app`, because it should be

Reply:
I think I'm with @NSExceptional on this one. If this is the commonly used directly, then we should keep it as-is.

Reply:
Cool, cool. Which leading slash? Did you mean `/path/to...` → `path/to...`?

Reply:
Yep, that's the one @NSExceptional :D

(Also, @github it would be very nice if it actually prompted for relevant usernames when I type `@`.....)

Reply:
Alright, good to go then! All the fixes are in the initial commit, which I force-pushed. Ready to merge if that's all.